File: test_fix.py
# 测试修复后的sendMessage函数逻辑

def test_send_message_logic():
    test_messages = [
        '@电影 https://example.com/movie',
        '@伯小爵 你好',
        '@音乐 周杰伦',
        '@天气 北京',
        '普通消息测试'
    ]
    
    print('=== 测试消息处理逻辑 ===')
    
    for message in test_messages:
        print(f'\n测试消息: {message}')
        
        try:
            # 模拟sendMessage函数的消息类型判断逻辑
            if message.startswith('@电影'):
                print('✅ 处理@电影消息')
            elif message.startswith('@伯小爵'):
                print('✅ 处理@伯小爵消息')
            elif message.startswith('@音乐'):
                print('✅ 处理@音乐消息')
            elif message.startswith('@天气'):
                print('✅ 处理@天气消息')
            else:
                print('✅ 处理普通消息')
        except Exception as e:
            print(f'❌ 错误: {e}')
    
    print('\n=== 测试完成 ===')

# 运行测试
if __name__ == '__main__':
    test_send_message_logic()

File: test_fix.js
// 测试修复后的sendMessage函数逻辑
function testSendMessageLogic() {
    const testMessages = [
        '@电影 https://example.com/movie',
        '@伯小爵 你好',
        '@音乐 周杰伦',
        '@天气 北京',
        '普通消息测试'
    ];
    
    console.log('=== 测试消息处理逻辑 ===');
    
    testMessages.forEach(message => {
        console.log(`\n测试消息: ${message}`);
        
        try {
            // 模拟sendMessage函数的消息类型判断逻辑
            if (message.startsWith('@电影')) {
                console.log('✅ 处理@电影消息');
            } else if (message.startsWith('@伯小爵')) {
                console.log('✅ 处理@伯小爵消息');
            } else if (message.startsWith('@音乐')) {
                console.log('✅ 处理@音乐消息');
            } else if (message.startsWith('@天气')) {
                console.log('✅ 处理@天气消息');
            } else {
                console.log('✅ 处理普通消息');
            }
        } catch (error) {
            console.log(`❌ 错误: ${error.message}`);
        }
    });
    
    console.log('\n=== 测试完成 ===');
}

// 运行测试
testSendMessageLogic();